### PR TITLE
Fix readme

### PR DESCRIPTION
**Pull request by michenly**
Fix a few quilt repo things.

### CODE REVIEW

File: templates/ROOT_README.hbs.md [hunk 1]
```diff
@@ -43,4 +43,4 @@ For Shopifolk, you can reach out to us in Slack in the `#web-foundation-tech` ch
 
 MIT &copy; [Shopify](https://shopify.com/), see [LICENSE.md](LICENSE.md) for details.
 
-<a href="http://www.shopify.com/"><img src="https://cdn.shopify.com/assets2/press/brand/shopify-logo-main-small-f029fcaf14649a054509f6790ce2ce94d1f1c037b4015b4f106c5a67ab033f5b.png" alt="Shopify" width="200" /></a>
+<a href="http://www.shopify.com/"><img src="https://cdn.shopify.com/assets2/brand-assets/shopify-logo-main-8ee1e0052baf87fd9698ceff7cbc01cc36a89170212ad227db3ff2706e89fd04.svg" alt="Shopify" width="200" /></a>
```
Comment (michenly) [inline templates/ROOT_README.hbs.md]: this need to be updated because of https://github.com/Shopify/quilt/pull/1139

File: README.md [hunk 2]
```diff
@@ -81,10 +82,13 @@ Each package has its own `README` and documentation describing usage.
 | react-testing | [directory](packages/react-testing) | [![npm version](https://badge.fury.io/js/%40shopify%2Freact-testing.svg)](https://badge.fury.io/js/%40shopify%2Freact-testing) |
 | react-tracking-pixel | [directory](packages/react-tracking-pixel) | [![npm version](https://badge.fury.io/js/%40shopify%2Freact-tracking-pixel.svg)](https://badge.fury.io/js/%40shopify%2Freact-tracking-pixel) |
 | react-universal-provider | [directory](packages/react-universal-provider) | [![npm version](https://badge.fury.io/js/%40shopify%2Freact-universal-provider.svg)](https://badge.fury.io/js/%40shopify%2Freact-universal-provider) |
+| react-web-worker | [directory](packages/react-web-worker) | [![npm version](https://badge.fury.io/js/%40shopify%2Freact-web-worker.svg)](https://badge.fury.io/js/%40shopify%2Freact-web-worker) |
+| rpc | [directory](packages/rpc) | [![npm version](https://badge.fury.io/js/%40shopify%2Frpc.svg)](https://badge.fury.io/js/%40shopify%2Frpc) |
```
Comment (michenly) [inline README.md]: cc @lemonmade just want to be sure these were not left off on purpose

File: .vscode/settings.json [hunk 3]
```diff
@@ -24,8 +24,8 @@
     "**/*.js": true
   },
   "editor.formatOnSave": true,
-  "prettier.stylelintIntegration": true,
-  "prettier.eslintIntegration": true,
+  "prettier-stylelint.stylelintIntegration": true,
```
Comment (michenly) [inline .vscode/settings.json]: cc @cartogram I had to update these two to get prettier working.
Screen Shot 2019-11-11 at 10 27 37 AM

double checking on if this is related to eslint update you did recently / if there are anything better to do

Comment (cartogram) [reply .vscode/settings.json]: Make sense to me 👍

File: README.md (hunk 2 above)
Comment (lemonmade) [reply README.md]: It's fine 👍

**Pull request merged**